SourceryTests/Stub/JavaScriptTemplates/Includes.js

```diff
@@ -0,0 +1 @@
+<%- include('Equality.js') %>-%><%- include('Other.js') %>-%>
```

Comment:
Yep, I know this tag but was not aware that it can be used alone. Actually it's now supported in swift too, but I guess just using closing tag will not work there...